### DISCUSSION TITLE
Do you plan to make Treeview?

### DISCUSSION THREAD

**Discussion by prawee** [mantinedev/mantine#223] [Q&A] [answered]
I want to display tree view  with categories support children.

thank you.

**Comment by rtivital** [comment, marked as answer]
Yes, at some point, not in one of next 2 releases though. If you want you can contribute such a component, there is not a ticket for this in trello board, but you can follow general instructions here – https://trello.com/c/htMs5VRI/1-new-component-cheklist

**Reply by prawee** [reply]
thank you for reply @rtivital  and big thank for best tools for save my time.

**Comment by ghost** [comment]
What's the status on this? Although there are quite many projects out there (with a broad range in terms of features & maintainer status), none seem to be a very good fit ..

**Reply by rtivital** [reply]
It is planned, no ETA.

**Reply by dextercai** [reply]
It has been a long time since the TreeView component been mentioned. Hopefully we will get to see this component to be released soon.
